diff --git a/package/fftw/fftw-single/fftw-single.mk b/package/fftw/fftw-single/fftw-single.mk
index 1e26ff7a..8b6b20c6 100644
--- a/package/fftw/fftw-single/fftw-single.mk
+++ b/package/fftw/fftw-single/fftw-single.mk
@@ -17,7 +17,9 @@ FFTW_SINGLE_CONF_ENV = $(FFTW_COMMON_CONF_ENV)
 FFTW_SINGLE_CONF_OPTS = \
 	$(FFTW_COMMON_CONF_OPTS) \
 	CFLAGS="$(FFTW_SINGLE_CFLAGS)" \
-	--enable-single
+	--enable-single \
+	--enable-static \
+	--disable-shared
 
 FFTW_SINGLE_CFLAGS = $(FFTW_COMMON_CFLAGS)
 

